recipes/tools/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
so this is the way we keep then synch?

Reply:
Right. Developers take care of this, and clients of recipes only care about a single clone that brings everything. It's a copy, but it's automated. The `deploy_*.sh` scripts were making it difficult to integrate with dancn' approach for portainer templating.